lib/hammer_cli_katello/id_resolver.rb

```diff
@@ -64,12 +64,18 @@ def environment_ids(options)
         return options['option_lifecycle_environment_ids']
       end
 
-      key_names = HammerCLI.option_accessor_name 'lifecycle_environment_names'
-      key_organization_id = HammerCLI.option_accessor_name 'organization_id'
-      options[key_organization_id] ||= organization_id(scoped_options 'organization', options)
+      if options["option_lifecycle_environment_names"]
+        key_names = HammerCLI.option_accessor_name "lifecycle_environment_names"
+      elsif options["option_environment_names"]
+        key_names = HammerCLI.option_accessor_name "environment_names"
+      elsif options["option_names"]
```

Comment:
I'm a bit concerned about use of the unscoped 'names' here. If there is a command `some-resource --names x,y` that has also optional param `--lifecycle-environment-names` it would lead the command to resolve --names as environment_ids. I'm not sure if such situation can happen though.